_contentTemplates/dropdowns/features.md

```diff
@@ -36,7 +36,7 @@ By virtualizing the elements in the dropdown, you can use huge data sources with
 | --- | --- | --- |
 | `AnimationDuration` | `int` | The animation duration of the popup in milliseconds. |
 | `Class` | `string` | Additional CSS class to customize the appearance of the popup. |
-| `Height` | `string` | The height of the popup. |
+| `Height` | `string` | The height of the popup. If set to **auto**, the component will automatically adjust the height of the popup based on the available visual viewport. However, it's important to note that if you set the `MaxHeight` property of the component or manually set the maximum height of the popup in any other way, the built-in boundary detection functionality will be disabled. This means the component will no longer automatically adjust to fit within the visible area of the viewport.|
```

Comment:
* Normally, we use **bold** only for names of interface elements. In this case, `"auto"` is inline string code.
* "manually set the maximum height of the popup in any other way" - this happens rarely, so I suggest that we don't over-complicate our explanation. People how have done it will know what to do.
* The viewport has no visible and invisible areas. The viewport is the frame of the browser that shows the web page, or a part of it.

```suggestion
| `Height` | `string` | The height of the popup. If set to `"auto"`, the component will automatically adjust the popup height based on the number of items and available space. Note that if you set the `MaxHeight` parameter, the built-in screen boundary detection will be disabled. This means the component will no longer adjust the drop down height to fit within the viewport. |
```